src/cljc/jinteki/preconstructed.cljc

```diff
@@ -0,0 +1,1520 @@
+           {:qty 2 :card "Funhouse"}
+           {:qty 2 :card "Superconducting Hub"}
+           {:qty 2 :card "Nico Campaign"}
+           {:qty 2 :card "Regolith Mining License"}
+           {:qty 2 :card "Urtica Cipher"}
+           {:qty 2 :card "Government Subsidy"}
+           {:qty 3 :card "Hedge Fund"}
+           {:qty 2 :card "Seamless Launch"}
+           {:qty 1 :card "Manegarm Skunkworks"}
+           {:qty 2 :card "Brân 1.0"}
+           {:qty 3 :card "Palisade"}
+           {:qty 2 :card "Diviner"}
+           {:qty 2 :card "Whitespace"}
+           {:qty 2 :card "Karunā"}
+           {:qty 2 :card "Tithe"}]})
+
+(def gateway-beginner-runner
+  {:format "system-gateway"
+   :identity {:title "The Catalyst: Convention Breaker" :side "Runner" :code 30076}
+   :name "System Gateway Starter Runner"
+   :cards [{:qty 2 :card "Creative Commission"}
+           {:qty 3 :card "Jailbreak"}
+           {:qty 2 :card "Overclock"}
+           {:qty 3 :card "Sure Gamble"}
+           {:qty 2 :card "Tread Lightly"}
+           {:qty 2 :card "VRcation"}
+           {:qty 1 :card "Docklands Pass"}
+           {:qty 1 :card "Pennyshaver"}
+           {:qty 1 :card "Red Team"}
+           {:qty 2 :card "Smartware Distributor"}
+           {:qty 2 :card "Telework Contract"}
+           {:qty 1 :card "Verbal Plasticity"}
+           {:qty 2 :card "Carmen"}
+           {:qty 2 :card "Cleaver"}
+           {:qty 2 :card "Mayfly"}
+           {:qty 2 :card "Unity"}]})
+
+(def gateway-intermediate-runner
+  {:format "system-gateway"
+   :identity {:title "The Catalyst: Convention Breaker" :side "Runner" :code 30076}
+   :name "System Gateway Starter Runner"
+   :cards [{:qty 2 :card "Creative Commission"}
+           {:qty 3 :card "Jailbreak"}
+           {:qty 2 :card "Overclock"}
+           {:qty 2 :card "Mutual Favor"}
+           {:qty 2 :card "Wildcat Strike"}
+           {:qty 2 :card "DZMZ Optimizer"}
+           {:qty 2 :card "Conduit"}
+           {:qty 2 :card "Leech"}
+           {:qty 3 :card "Sure Gamble"}
+           {:qty 2 :card "Tread Lightly"}
+           {:qty 2 :card "VRcation"}
+           {:qty 1 :card "Docklands Pass"}
+           {:qty 1 :card "Pennyshaver"}
+           {:qty 1 :card "Red Team"}
+           {:qty 2 :card "Smartware Distributor"}
+           {:qty 2 :card "Telework Contract"}
+           {:qty 1 :card "Verbal Plasticity"}
+           {:qty 2 :card "Carmen"}
+           {:qty 2 :card "Cleaver"}
+           {:qty 2 :card "Mayfly"}
+           {:qty 2 :card "Unity"}]})
+
+(def system-gateway-beginner
+  (matchup
+    [:lobby.gateway-format.beginner "Beginner"]
+    [:lobby.gateway-format.beginner "Beginner"]
+    [:lobby.gateway-format.beginner-info
+     "This lobby is using the System Gateway beginner decks for the Corporation and Runner. These decks are recommended for your first games. Games are played to 6 agenda points."]
+    [:lobby.gateway-format-beginner-ul "System Gateway - Beginner Teaching Decks"]
```

Comment:
None of these `-ul` are used anywhere or defined anywhere, and they're all duplicating other info. What's the purpose?

Reply:
This and the others are also `:lobby.gateway-format-beginner-ul` instead of matching the other keys with `:lobby.gateway-format.beginner-ul`.